packages/ui/src/styles/global.scss

```diff
@@ -1,2 +1 @@
 @import "./base";
-@import "./components";
```

Comment:
With these changes, [this](https://faststore-site-git-feat-fs-778-creates-navbar-71017f-faststore.vercel.app/components#all-components-included) won't be true anymore, right?

<img width="592" alt="image" src="https://user-images.githubusercontent.com/3356699/234079466-f3f29e06-d20f-4bac-b03e-c1a45aac3b81.png">

So, we should change to 

```
All components included
@import "@faststore/ui/src/styles/components.scss";
```

Is that right?

Reply:
Nice catch! Makes sense to me. Done [here](https://github.com/vtex/faststore/pull/1712/commits/986405809f9e487bb56331a321f52577a5970bb0).